config/settings.d/logs.yml.example

```diff
@@ -0,0 +1,5 @@
+---
+# Can be true, false, or http/https to enable just one of the protocols
+:enabled: false
+
+# Log buffer configuration is in core file (settings.yml)
```

Comment:
This config file deals with log api only (exposing events over http). This module may or may not be enabled, but internal logging is always available.

I think the main settings file is a good spot for these.